### DISCUSSION TITLE
Idée Addon pour Scratch Addons

### DISCUSSION THREAD

**Discussion by ProScratcheurPro** [ScratchAddons/ScratchAddons#5944] [Ideas] [answered]
J'aurais un idée d'addon à ajouter ; ce serait d'ajouter la possibilité de faire tourner un fond d'écran à l'aide d'un bloc de rotation dans la liste des blocs dans la partie ''code'' des fond d'écran pour chaque fond d'écran.
Merci d'y réfléchir.😃

ProScratcheurPro

**Comment by BroJac5246** [comment, marked as answer]
Merci pour l'idée ! Cependant, nous ne pouvons pas ajouter d'addons qui ajoutent des fonctionnalités qui casseraient Scratch pour les utilisateurs réguliers. Étant donné que Scratch n'autorise pas la rotation de la scène, les projets qui l'utilisent seraient interrompus pour les utilisateurs réguliers. Pour cette raison, nous ne pouvons pas ajouter ceci.

(Traduit à l'aide de Google Traduction)

_-English translation below-_

Thanks for the idea! However, we cannot add addons that add functionality that would break Scratch for regular users. Since Scratch does not allow the stage to be rotated, projects using this would break for regular users. Because of that, we can't add this.

(Translated [into French] using Google Translate)

**Reply by ProScratcheurPro** [reply]
Thanks for answering my idea.😁

**Reply by ProScratcheurPro** [reply]
It's ok.

**Comment by ProScratcheurPro** [comment]
Désolé encore de vous déranger, je n'ai pas été clair. D'ailleurs, il y a plutôt un problème avec les sprites qui tournent et qui sont censés occuper tout l'espace, mais quand ils tournent, ils n'occupent pas les coins et ils se transforment en rectangles tournants à la place.

Bref, je vous laisse un lien vers un de mes projets pour vous montrer parce que ça ne se décrit pas bien.
Voici le lien : [https://scratch.mit.edu/projects/838115055/editor/](https://scratch.mit.edu/projects/838115055/editor/)
Faites juste cliquer sur le drapeau vert et regarder la partie du sprite qui ne s'affiche pas.
Je voudrais voir les rayons oranges s'étendre jusqu'au bord de la scène.
Vous pourriez faire un addons pour régler ça?

Merci, sincèrement
ProScratcheurPro

**Reply by BroJac5246** [reply]
That's caused by Scratch attempting to stop the sprite from exiting the stage. We probably shouldn't fix that here because it might break some projects' functionality for our users, and our users may make projects that break on plain Scratch. It's a known bug (feature?), but it could be reported on [Scratch's repository](https://github.com/LLK/scratch-gui).

_-Translation-_

Cela est dû au fait que Scratch tente d'empêcher le sprite de quitter la scène. Nous ne devrions probablement pas corriger cela ici car cela pourrait casser la fonctionnalité de certains projets pour nos utilisateurs, et nos utilisateurs peuvent créer des projets qui se cassent sur Scratch ordinaire. C'est un bogue connu (fonctionnalité ?), mais il pourrait être signalé sur le[ référentiel de Scratch](https://github.com/LLK/scratch-gui).

(Traduit via Google Traduction)

**Reply by ProScratcheurPro** [reply]
Ok, merci.
